mithril-test-lab/mithril-devnet/mkfiles/mkfiles-init.sh

```diff
@@ -1,9 +1,18 @@
+# Get the operating system
+UNAME=$(uname -s) OPERATING_SYSTEM=
```

Comment:
Is it necessary to put this 2 commands in the same line ? 
This can be more complicated to understand than having each instruction on one line.

Reply:
I saw it was already done this way in this [file](https://github.com/input-output-hk/mithril/blob/main/mithril-test-lab/mithril-devnet/mkfiles/mkfiles-cardano.sh). I guessed it was ok